quickstarts/Video.ipynb

```diff
@@ -114,7 +114,9 @@
         "\n",
```

Comment:
<p>Nit: link to your upload files notebook, instead of just the examples folder.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/41/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Done